src/trapfunc.cpp

```diff
@@ -808,7 +808,7 @@ bool trapfunc::pit( const tripoint &p, Creature *c, item * )
             you->add_msg_if_player( _( "You flap your wings and flutter down gracefully." ) );
         } else if( you->has_active_bionic( bio_shock_absorber ) ) {
```

Comment:
Can you un-hardcode the functionality instead? Like by maybe moving it to a flag.

Reply:
Honestly it's not worth it, we can just wait for a second user of it, and fully expect that one to never appear.